tests/Propel/Tests/Generator/Builder/Om/QueryBuilderInheritanceTest.php

```diff
@@ -137,25 +152,42 @@ public function testFindPkSimpleWithSingleTableInheritanceReturnCorrectClass()
         $cashier2 = new BookstoreCashier();
         $cashier2->save($this->con);
 
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreEmployee', BookstoreEmployeeQuery::create()->findPk($employee->getId()),
-            'findPk() return right object : BookstoreEmployee');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreManager', BookstoreEmployeeQuery::create()->findPk($manager->getId()),
-            'findPk() return right object : BookstoreManager');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreCashier', BookstoreEmployeeQuery::create()->findPk($cashier1->getId()),
-            'findPk() return right object : BookstoreCashier');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreCashier', BookstoreEmployeeQuery::create()->findPk($cashier2->getId()),
-            'findPk() return right object : BookstoreCashier');
+        $this->assertInstanceOf(
+            '\Propel\Tests\Bookstore\BookstoreEmployee',
```

Comment:
can we use BookstoreEmployee::class here and in in other places in this method?

Reply:
Next sniffer run, there are more to come (with manual ones).